src/i18n/pt-BR/ui.ts

```diff
@@ -9,8 +18,17 @@ export default UIDictionary({
 	'rightSidebar.editPage': 'Edite esta página',
 	'rightSidebar.translatePage': 'Traduza esta página',
 	'rightSidebar.joinCommunity': 'Junte-se à comunidade',
+	'rightSidebar.a11yTitle': 'Índice',
 	'articleNav.nextPage': 'Próxima página',
 	'articleNav.prevPage': 'Página anterior',
 	'since.addedIn': 'Adicionado em:',
 	'since.new': 'Novo',
+	'install.autoTab': 'Interface de Linha de Comando Automática',
+	'install.manualTab': 'Instalação Manual',
```

Comment:
Should we translate the word `setup` as `Configuração`?

Reply:
In this specific case I think `Instalação` works better since this tab is about installing Astro manually, but yeah, sometimes `Configuração` is better. I just tend to associate it more with changing settings than with adding/installing/creating something and because of that, I opted for `Instalação`.

Thanks for all the reviews ❤️

Reply:
I understand! `Instalação` seems to fit better in this case as the first step to obtain Astro. I was asking because I think it's important to find out a consistent translation throughout the docs, I'll keep my head up to your explanation as I work on the other pages!

Thanks!